sites/fast-website/src/app/components/content-placement/content-placement.ts

```diff
@@ -0,0 +1,5 @@
+import { FASTElement, attr } from "@microsoft/fast-element";
+
+export class ContentPlacement extends FASTElement {
+    @attr divider: boolean = false;
```

Comment:
you'll want to do:
```
@attr({ mode: "boolean" })
public divider = true;
```
this will keep the attribute from writing to the DOM when its false

Reply:
Will do! Thank you.